README.md

```diff
@@ -1,73 +1,69 @@
 # Project Name
 
-> One paragraph statement about the project.
-
-![screenshot](./app_screenshot.png)
+> We have created a command line TicTacToe Project using Ruby.
 
 Additional description about the project and its features.
 
 ## Built With
 
-- Major languages,
-- frameworks,
-- technologies used
-
-## Live Demo
+- Ruby,
+- Vs.code,
+- Git and Github
 
-[Live Demo Link](https://livedemo.com)
+## Getting Started
 
+- clone this repository `https://github.com/rajkumardas2701/Tic-Tac-Toe`
+- cd to the folder `Tic-Tac-Toe`
+- To run the application use this command `ruby bin/main.rb`
```

Comment:
And after this?
- [ ] Explicitly tell the user how your game interface works and the commands they will have to input. Be explicit about it being a terminal/command prompt game.

- [ ] [OPTIONAL] Add a screenshot or code snippet showing the game UI